wagtail/admin/forms/workflows.py

```diff
@@ -1,9 +1,77 @@
 from django import forms
 from django.core.exceptions import ValidationError
-from django.utils.translation import ugettext_lazy as _
+from django.utils.functional import cached_property
+from django.utils.translation import ugettext as _
+from django.utils.translation import ugettext_lazy as __
 
 from wagtail.admin import widgets
-from wagtail.core.models import Page, Workflow, WorkflowPage
+from wagtail.core.models import Page, Task, Workflow, WorkflowPage
+
+
+def model_name(model):
+    return model._meta.app_label + '.' + model.__name__
```

Comment:
could this live in `core.utils`? especially that it contains `resolve_model_string` which is the opposite of this, in a way